server/service/mdm.go

```diff
@@ -2141,6 +2141,10 @@ func (svc *Service) GetMDMAppleCSR(ctx context.Context) ([]byte, error) {
 		return nil, err
 	}
 
+	if len(svc.config.Server.PrivateKey) == 0 {
+		return nil, ctxerr.Wrap(ctx, errors.New("no private key configured"))
```

Comment:
```suggestion
		return nil, ctxerr.New(ctx, "no private key configured")
```

Reply:
sanity checking: this will result in a `500` error, which might be fine! just making sure if it's intentional

Reply:
Yeah, that sounds right to me (since the server is not properly configured).